tests/test_route.py

```diff
@@ -524,21 +490,19 @@ def test_RouteAndNexthopInDifferentVrf(self, dvs, testlog):
         time.sleep(1)
 
         # check application database
-        tbl = swsscommon.Table(self.pdb, "ROUTE_TABLE:Vrf_1")
-        route_entries = tbl.getKeys()
+        route_entries = self.pdb.get_keys("ROUTE_TABLE:Vrf_1")
         assert "20.0.1.2" in route_entries
 
-        tbl = swsscommon.Table(self.pdb, "ROUTE_TABLE:Vrf_2")
-        route_entries = tbl.getKeys()
+        route_entries = self.pdb.get_keys("ROUTE_TABLE:Vrf_2")
         assert "10.0.0.2" in route_entries
 
         # check ASIC neighbor interface database
-        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_NEXT_HOP")
-        nexthop_entries = tbl.getKeys()
+        nexthop_entries = self.adb.get_keys("ASIC_STATE:SAI_OBJECT_TYPE_NEXT_HOP")
         for key in nexthop_entries:
-            (status, fvs) = tbl.get(key)
+            fvs = self.adb.get_entry("ASIC_STATE:SAI_OBJECT_TYPE_NEXT_HOP", key)
+            status = bool(fvs)
             assert status == True
-            for fv in fvs:
+            for fv in list(fvs.items()):
```

Comment:
>fvs [](start = 27, length = 3)

Could you use fvs as a dict, instead of converting back to list?

Reply:
Fixed this and two later ones.